### PR TITLE
PR for #5320OpensslEngine throws "session id context uninitialized" when used with client authentication

### PR DESCRIPTION
**Pull request by floragunncom**
set the session id context properly

### CODE REVIEW

File: handler/src/main/java/io/netty/handler/ssl/OpenSslServerContext.java [hunk 1]
```diff
@@ -439,6 +439,7 @@ void verify(OpenSslEngine engine, X509Certificate[] peerCerts, String auth)
                 }
             }
             sessionContext = new OpenSslServerSessionContext(ctx);
+            sessionContext.setSessionIdContext("netty".getBytes());
```
Comment (normanmaurer) [inline handler/src/main/java/io/netty/handler/ssl/OpenSslServerContext.java]: Please define a static byte[] and reuse everytime. Something like:

private static final byte[] ID = new byte[] {'n', 'e', 't', 't', 'y'};

**Comment by normanmaurer** [comment]
@floragunncom thanks! Could you please adjust the commit message to match our template: 

http://netty.io/wiki/writing-a-commit-message.html

Please once done, squash the commits and let me know

**Comment by floragunncom** [comment]
done

**Comment by normanmaurer** [comment]
Did you sign the icla yet?

http://netty.io/s/icla

**Comment by floragunncom** [comment]
just signed the icla

**Comment by normanmaurer** [comment]
Cherry-picked into 4.0 (87efddcda789a993b8acd74dac8665305af127b8) and 4.1 (528b83e27738af887955665a0ba82e081c7d8729)

**Comment by normanmaurer** [comment]
@floragunncom thanks!

**Comment by floragunncom** [comment]
whats the ETA for 4.0.37.Final?

**Comment by normanmaurer** [comment]
Next week

**Pull request closed without merging**